synapse_invite_checker/invite_checker.py

```diff
@@ -404,44 +409,100 @@ async def is_domain_allowed(self, domain: str) -> bool:
     async def is_domain_insurance(self, domain: str) -> bool:
         return await self._domain_list_check(lambda fl: fl.is_insurance(domain))
 
+    async def on_create_room(
+        self,
+        requester: Requester,
+        request_content: dict[str, Any],
+        is_request_admin: bool,
+    ) -> None:
+        """
+        Raise a SynapseError if creating a room with invites should be denied
+        """
+        # Unlike `user_may_invite()`, `on_create_room()` runs with the inviter being
+        # a local user and the invitee is remote
+        inviter = requester.user
+        assert self.api.is_mine(
+            inviter.to_string()
+        ), "creating a room for a user that wasn't from this server is strange"
+
+        invites_block: list[str] = request_content.get("invite", [])
```

Comment:
```suggestion
        invite_list: list[str] = request_content.get("invite", [])
```

No idea, why that would be a block.

Reply:
no idea either, something I missed at the time. Fixed now